src/web/impl/WsBase.hpp

```diff
@@ -181,8 +174,12 @@ class WsBase : public ConnectionBase, public std::enable_shared_from_this<WsBase
         boost::asio::dispatch(
             derived().ws().get_executor(),
             [this, self = derived().shared_from_this(), msg = std::move(msg)]() {
+                if (messages_.size() > maxSendingQueueSize_) {
+                    wsFail(boost::asio::error::timed_out, "Client is too slow");
```

Comment:
Should we change `wsFail()` to log such errors in webserver's log?

Reply:
Good idea

Reply:
We 'd better not to remove the perf log. Maybe we can leave it as it is, since it is error level.